package.json

```diff
@@ -52,9 +52,14 @@
   },
   "dependencies": {
     "axios": "^0.19.0",
+    "chalk": "^2.4.2",
```

Comment:
Are any of these dependencies necessary for this fix?

Reply:
They are not necessary for this fix, but I included it because peer dependency warnings can be potentially problematic at some point when required, but never installed. I can undo the changes but keep it in mind ;)